src/style.css

```diff
@@ -1 +1,42 @@
+/* stylelint-disable no-descending-specificity */
+
 /* add styles here */
+body {
+display: flex;
+flex-direction: column;
+align-items: center;
+margin: 0;
+}
+
+.stars {
+display: flex;
+}
+
+
+
+.stars--5 .stars__star,
+.stars--1 > .stars__star,
+.stars--2 > .stars__star:nth-child(-n+2),
+.stars--3 > .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:not(:last-child) {
+  width: 16px;
+  height: 16px;
+  background-image: url('./images/star-active.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star,
+.stars--1 .stars__star:not(:first-child),
+.stars--2 .stars__star:not(:nth-child(-n+2)),
+.stars--3 .stars__star:not(:nth-child(-n+3)),
+.stars--4 .stars__star:last-child {
+  width: 16px;
+  height: 16px;
+  background-image: url('./images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
```

Comment:
this styles should be refactored:
1. use `nth-child` only for setting background-image and combine it for all at once with comma
2. move other styles to `start__star` class